love/src/components/Facility/Map/Device.jsx

```diff
@@ -0,0 +1,366 @@
+                  <tspan>{fixedFloat(parameters[x].params[ctx].value, 2) + parameters[x].params[ctx].unit}</tspan>
+                </text>
+              </React.Fragment>
+            );
+          }
+
+          //Box type group parameter
+          else if (parameters[x].params[ctx].type === 'box') {
+            return (
+              <React.Fragment>
+                <rect
+                  className={this.statusTextToStyle(String(parameters[x].params[ctx].state))}
+                  width={10}
+                  height={10}
+                  transform={'translate(' + (transX - 5) + ' 0)'}
+                />
+                <text className={styles.paramVal} transform={'translate(' + transX + ' 6)'} textAnchor="middle">
+                  <tspan>{parameters[x].params[ctx].name}</tspan>
+                </text>
+                <text className={styles.paramVal} transform={'translate(' + transX + ' 15)'} textAnchor="middle">
+                  <tspan>
+                    {parameters[x].params[ctx].value
+                      ? fixedFloat(parameters[x].params[ctx].value, 2) + ' ' + parameters[x].params[ctx].unit
+                      : ''}
+                  </tspan>
+                </text>
+              </React.Fragment>
+            );
+          }
+
+          //noBox type group parameter
+          else {
+            return (
+              <React.Fragment>
+                <text className={styles.paramVal} transform={'translate(' + transX + ' 7)'} textAnchor="middle">
+                  <tspan>{parameters[x].params[ctx].name}</tspan>
+                </text>
+                <text className={styles.paramVal} transform={'translate(' + transX + ' 13)'} textAnchor="middle">
+                  <tspan>{fixedFloat(parameters[x].params[ctx].value, 2) + parameters[x].params[ctx].unit}</tspan>
+                </text>
+              </React.Fragment>
+            );
+          }
+        });
+
+        return (
+          <g transform={'translate(0 ' + i + ')'}>
+            <text className={styles.paramGroup} transform={'translate(4 0)'} textAnchor="start">
+              <tspan>{parameters[x].name}</tspan>
+            </text>
+            <text className={styles.paramVal} transform={'translate(' + (width - 4) + ' 0)'} textAnchor="end">
+              <tspan>
+                {(parameters[x].value ? fixedFloat(parameters[x].value, 2) : '') +
+                  (parameters[x].unit ? parameters[x].unit : '')}
+              </tspan>
+            </text>
+            <g transform={'translate(4 4)'}>{groupBox}</g>
+            <g transform={'translate(4 10)'}>{groupData}</g>
+          </g>
+        );
+      }
+    });
+  }
+
+  render() {
+    const { title, height, temp, width, scale, posX, posY, collapsible } = this.props;
+    const hidden = this.state.hidden;
+    const isAlarmed = Object.keys(this.props.alarms).some((a) => {
+      return this.props.alarms[a].state;
+    });
+    var mask = this.id;
```

Comment:
I would suggest to use just `this.id` directly instead of assigning it to a new `mask`var. Please, at least set it as:
```javascript
const mask = this.id;
```